### PR TITLE
[WFLY-9800]:  ASM version divergence after org.apache.cxf:cxf-rt-bindings-xml version upgrade

### PR DESCRIPTION
**Pull request by ehsavoie**
Excluding asm:asm and inherited dependency on org.ow2.asm:asm and defining a global dependency on org.ow2.asm:asm.

Jira: https://issues.jboss.org/browse/WFLY-9800

@asoldano please review

### CODE REVIEW

File: feature-pack/src/main/resources/modules/system/layers/base/org/ow2/asm/asm/main/module.xml [hunk 1]
```diff
@@ -22,13 +22,13 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<module xmlns="urn:jboss:module:1.5" name="asm.asm">
+<module xmlns="urn:jboss:module:1.5" name="org.ow2.asm.asm">
```
Comment (ctomc) [inline feature-pack/src/main/resources/modules/system/layers/base/org/ow2/asm/asm/main/module.xml]: I think module should be named only org.ow2.asm, without second asm repeated.
or we could even stay with old name.

but beyond that it looks ok.

**Review by ctomc — APPROVED** [review]

File: feature-pack/src/main/resources/modules/system/layers/base/org/ow2/asm/asm/main/module.xml (hunk 1 above)
Comment (ehsavoie) [reply feature-pack/src/main/resources/modules/system/layers/base/org/ow2/asm/asm/main/module.xml]: That was my first idea, then I saw that there were several artifacts under the group id org.ow2.asm so i wanted to be able to add those

**Review by asoldano — APPROVED** [review]
As Tomaz mentioned, we could have gone with the same module name as before, but I have no strong opinion on this, so I'm fine. I've tested jbossws-cxf testsuite with asm 6 locally and it works as expected, so definitely +1 on this. Thanks.

**Comment by asoldano** [comment]
In any case, the failure in TC seems related to the change here.

**Pull request merged**